insights-notification-schemas-java/src/main/resources/schemas/Action-out.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$comment": "This schema defines some additional information for an Action that might be useful",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "allOf": [
+    {
+      "$ref": "./Action.json"
+    }
+  ],
+  "description": "Describes additional information fields for some of the fields of the Action",
+  "properties": {
+    "source": {
+      "additionalProperties": false,
+      "application": {
+        "additionalProperties": false,
+        "description": "Represents the extra information for the application object",
+        "display_name": {
+          "description": "A user friendly version of the application's name",
+          "title": "The application's display name",
+          "type": "string"
+        },
+        "title": "Application",
+        "type": "object"
+      },
+      "bundle": {
+        "additionalProperties": false,
+        "description": "Represents the extra information for the bundle object",
+        "properties": {
+          "display_name": {
+            "description": "A user friendly version of the bundle's name",
+            "title": "Bundle's display name",
+            "type": "string"
+          }
+        },
+        "title": "Bundle",
+        "type": "object"
+      },
+      "description": "The extended details related to the Action",
+      "event_type": {
+        "additionalProperties": false,
+        "description": "Represents the extra information for the event type object",
+        "properties": {
+          "display_name": {
+            "description": "A user friendly version of the event type's name",
+            "title": "Event type's display name",
+            "type": "string"
+          }
+        },
+        "title": "Event Type",
+        "type": "object"
+      },
+      "title": "Source",
+      "type": "object"
+    }
+  },
+  "title": "Action's details",
+  "type": "object"
```

Comment:
Could you move the `type` and `title` to the top or at least before the properties?
It would make them more readable (IMO)

Reply:
Yeah, I've moved all of them up to be more consistent